src/Panel/IncludePanel.php

```diff
@@ -137,16 +170,31 @@ protected function _isPluginFile($file)
     }
 
     /**
-     * Replace the path with APP, CORE or the plugin name
+     * Check if a path is from a Composer package
+     *
+     * @param string $file File to check
+     * @return bool
+     */
+    protected function _isVendorFile($file)
+    {
+        foreach ($this->_composerPaths as $package => $path) {
+            if (strstr($file, $path)) {
+                return $package;
+            }
+        }
+
+        return false;
+    }
+
+    /**
+     * Replace the path with APP, CAKE, ROOT or the plugin name
      *
      * @param string $file File to check
      * @param string $type The file type
-     *  - app for app files
-     *  - cake for cake files
-     *  - PluginName for the name of a plugin
+     * @param string $name plugin name or composer package
      * @return bool
      */
-    protected function _niceFileName($file, $type)
+    protected function _niceFileName($file, $type, $name = null)
```

Comment:
no need for name to be nullable

Reply:
Function gets 2 arguments when `$type` is app/cake/root, `$name` is used when type is plugin/vendor.

Reply:
Then you should fix the doc block.